src/components/LinkWrapper/LinkWrapper.jsx

```diff
@@ -0,0 +1,69 @@
+import { forwardRef } from 'react';
+import { MenuItem, StyledLink, StyledNavLink } from './styles';
+
+const LinkWrapper = forwardRef(function LinkWrap(props, ref) {
+  return (
+    <>
+      {!props.item.path && (
```

Comment:
Could you please explain what is path?

Reply:
Is it a tricky one? Path property was suposed to be an absolute path for given link/component